.github/workflows/test-and-publish-nightly.yml

```diff
@@ -51,64 +51,100 @@ jobs:
             java-version: 11
             java-distribution: corretto
             container: "public.ecr.aws/async-profiler/asprof-builder-arm:latest"
+          - runson:
+              display: amazonlinux-2023 # Use dash instead of colon, because GHA artifacts cannot contain colons
+              name: ubuntu-latest
```

Comment:
Do we need `ubuntu` in the name, when `container` is set to amazonlinux?

Reply:
Yes, this is the name of the runner, which then runs a Docker container from an `amazonlinux` image